src/schema/types/index.ts

```diff
@@ -1,4 +1,11 @@
-export * from "./user";
-export * from "./portfolio";
-export * from "./project";
-export * from "./skill";
+import { Mutation } from "./Mutation";
+import { Query } from "./Query";
+import * as Input from './inputs' ;
+import * as Model from './models' ;
+
+export const resolvers = {
+    Mutation,
+    Query,
+    ...Input,
+    ...Model
+}
```

Comment:
import 과정을 다음과 같이 생략할 수 있습니다.
```suggestion
export { Mutation } from "./Mutation";
export { Query } from "./Query";
export *  from './inputs' ;
export *  from './models' ;